### PR TITLE
DOC: Add space after argument name

### PR DESCRIPTION
**Pull request by oscargus**
Some are private, but still worthwhile to get it correct as some editors render properly.

Also fixed LaTeX formatting of cos and added link.

### CODE REVIEW

File: numpy/lib/function_base.py [hunk 1]
```diff
@@ -4713,10 +4713,10 @@ def trapz(y, x=None, dx=1.0, axis=-1):
...
     -------
     trapz : float or ndarray
-        Definite integral of 'y' = n-dimensional array as approximated along
-        a single axis by the trapezoidal rule. If 'y' is a 1-dimensional array,
-        then the result is a float. If 'n' is greater than 1, then the result
-        is an 'n-1' dimensional array.
+        Definite integral of `y` = n-dimensional array as approximated along
+        a single axis by the trapezoidal rule. If `y` is a 1-dimensional array,
+        then the result is a float. If `n` is greater than 1, then the result
+        is an ´n`-1 dimensional array.
```
Comment (charris) [inline numpy/lib/function_base.py:4719]: Apostrophe instead of back tick.

File: numpy/lib/function_base.py [hunk 2]
```diff
@@ -4847,7 +4847,7 @@ def meshgrid(*xi, copy=True, sparse=False, indexing='xy'):
     Returns
     -------
     X1, X2,..., XN : ndarray
-        For vectors `x1`, `x2`,..., 'xn' with lengths ``Ni=len(xi)`` ,
+        For vectors `x1`, `x2`,..., `xn` with lengths ``Ni=len(xi)`` ,
         return ``(N1, N2, N3,...Nn)`` shaped arrays if indexing='ij'
```
Comment (charris) [inline numpy/lib/function_base.py:4851]: return <- returns

File: numpy/lib/function_base.py [hunk 3]
```diff
@@ -2978,11 +2978,11 @@ def bartlett(M):
     processing literature, where it is used as one of many windowing
     functions for smoothing values.  Note that convolution with this
     window produces linear interpolation.  It is also known as an
-    apodization (which means"removing the foot", i.e. smoothing
+    apodization (which means "removing the foot", i.e. smoothing
     discontinuities at the beginning and end of the sampled signal) or
-    tapering function. The fourier transform of the Bartlett is the product
-    of two sinc functions.
-    Note the excellent discussion in Kanasewich.
+    tapering function. The Fourier transform of the Bartlett window is the
```
Comment (charris) [inline numpy/lib/function_base.py:2983]: Might refactor to better equalize line lengths for terminal display.

File: numpy/lib/format.py [hunk 4]
```diff
@@ -421,7 +421,7 @@ def _write_array_header(fp, d, version=None):
     d : dict
         This has the appropriate entries for writing its string representation
         to the header of the file.
-    version: tuple or None
+    version : tuple or None
         None means use oldest that works
```
Comment (charris) [inline numpy/lib/format.py:425]: Needs a trailing comma. Might equalize line lengths a bit better.

**Comment by charris** [comment]
Couple of nits, otherwise looks good.

File: numpy/lib/function_base.py (hunk 1 above)
Comment (oscargus) [reply numpy/lib/function_base.py]: Whoops!

**Comment by oscargus** [comment]
Fixed these things. For the trailing comma, it wasn't really clear to me what the sentence was about to say. (I didn't really check the surrounding texts when adding spaces.) I think the updated formulation is better (and hopefully correct...).

**Comment by charris** [comment]
Thanks @oscargus .

**Comment by oscargus** [comment]
Thanks for additional changes. I agree on them for sure, but only had my eyes on specific things...

**Pull request merged**